core/network/src/main/kotlin/nl/q42/template/core/network/di/NetworkModule.kt

```diff
@@ -43,12 +39,15 @@ internal class NetworkModule {
     @Provides
     fun provideRetrofit(
         httpClient: OkHttpClient,
-        moshi: Moshi,
         @ConfigApiMainPath apiMainPath: String,
     ): Retrofit {
+        val contentType = "application/json".toMediaType()
+
+        val json = Json { ignoreUnknownKeys = true }
```

Comment:
I think it could help other developers using this template for a project, to have some documentation directly here about this decision, in case they need to use JSON parsing a little bit differently.
What do you think?